bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/thing/device.xml

```diff
@@ -31,6 +31,7 @@
 				<option value="LOW_BATTERY">@text/channel-type.shelly.alarmTrigger.option.LOW_BATTERY</option>
 				<option value="BATTERY">@text/channel-type.shelly.alarmTrigger.option.BATTERY</option>
 				<option value="POWERON">@text/channel-type.shelly.alarmTrigger.option.POWERON</option>
+				<option value="EXT_POWER">@text/channel-type.shelly.alarmTrigger.option.EXT_POWER</option>
```

Comment:
This entry is missing in the properties.

Reply:
fixed